src/contact.ts

```diff
@@ -292,16 +378,31 @@ export class Contact implements Sayable {
   }
 
   /**
-   * get the alias for contact
+   * Get the alias for contact
+   *
+   * @returns {(string | null)}
+   *
+   * @memberOf Contact
```

Comment:
`@memberOf` is not needed in ES6/Typescript.

Because the JsDoc is already known it is member of the class.